File: meal_price/meal_price.c
#include <stdio.h>
/**
 * main - Compute the price of a meal as follows by asking for the price of 
 * child and adult meals, the number of each, and then the sales tax rate.
 * Use these values to determine the total price of the meal.
 * Then, ask for the payment amount and compute the amount of change
 * to give back to the customer.
 * Return: 0 always
 */
int main(void)
{
	float price_of_child;
	float price_of_adult;
	float num_children;
	float num_adult;
	float tax_rate;
	float sum_children;
	float sum_adult;
	float subtotal;
	float st;
	float sales_tax;
	float total;
	float payment;
	float change;

	/** taking and storing input from user */
	printf("\nenter the price of a child's meal: ");
	scanf("%f", &price_of_child);

	printf("\nenter the price of an adult meal: ");
	scanf("%f", &price_of_adult);

	printf("\nhow many children? ");
	scanf("%f", &num_children);
	
	printf("\nhow many adult? ");
	scanf("%f", &num_adult);

	printf("\nwhat is the sales tax rate? ");
	scanf("%f", &tax_rate);

	/** calculating for the sub total of all meal*/
	sum_children = price_of_child * num_children;
	sum_adult = price_of_adult * num_adult;
	subtotal = sum_children + sum_adult;
	printf("\nsubtotal: #%.2f\n", subtotal);

	/** •	Compute and display the sales tax,*/
	
	st = ((tax_rate) / (float) 100);
	sales_tax = subtotal * st;
	printf("sales tax: #%.2f\n", sales_tax);

	/** * •    Compute and display the total price of the meal.*/
	total = subtotal + sales_tax;
	printf("total cost: #%.2f\n", total);

	/*•	Ask the user for the the payment amount */

	printf("how much are you paying? ");
	scanf("\n%f", &payment);

	/*•	Compute and display the change.*/
	
	change = payment - total;
	printf("change: #%.2f\n", change);
	
	return 0;
}

File: BYU_CSE110/meal_price.py
                #creativity
'''# i added the calculation of the user's total monthly and yearly expenses, if he/she makes the purchase everyday.'''
                
                # the input 
C_M= float(input('what is the price of a childs meal? '))
A_M= float(input('what is the price an adult meal? '))
n_C= int(input('what is the number of children? '))
n_A= int(input('what is the number of adult? '))
S_T_R= float(input('what is the sales tax rate? '))
                # calculating the sub total
T_A= n_C * C_M
T_C= n_A * A_M
Sub_T=  T_A + T_C
sd= '${:,.2f}'.format(Sub_T)  #to display in dollar sign * '${:,.2f}'*
                # output
print(f'\nsubtotal:',sd) #sub total
sales_t= Sub_T * S_T_R     
rate= sales_t / 100
rd='${:,.2f}'.format(rate)
print(f'sales tax:',rd)  # sales tax
T_P= Sub_T + rate
tpd='${:,.2f}'.format(T_P)
print('total price',tpd)   #total price
                # payment amount
P_A= float(input('\nhow much are you paying? '))
change= P_A - T_P
change_d='${:,.2f}'.format(change)
print('your change is',change_d)   #change  

                #creativity
mon=T_P * 30 
mon_D='${:,.2f}'.format(mon)
year=mon * 12
year_D='${:,.2f}'.format(year)
print ('\nif you make this purchase everyday your expenditure:')
print('monthly:',mon_D)
print('yearly:',year_D)
